.github/workflows/phpunit.yml

```diff
@@ -25,7 +25,6 @@ jobs:
     strategy:
       matrix:
         php-version:
-          - "7.2"
```

Comment:
So, the minimum PHP version for v9 is 7.3?

Reply:
I'm going off of the composer.json: https://github.com/concrete5/concrete5/blob/9.0rc/concrete/composer.json#L28